### PR TITLE
Feat: Above-the-fold changes

### PR DESCRIPTION
**Pull request by flavioriper**
### **Description**
We need this to change the Tagline texts, change the secondary CTA to open a hubspot form dialog and the inclusion of a bar at the top of the page for an interactive tour.

### CODE REVIEW

File: src/components/SectionOne.tsx [hunk 1]
```diff
@@ -103,36 +102,35 @@ const SectionOne = () => {
...
 
 
     return (
         <div className="section-one">
             <div className="section-one-wrapper">
                 <div className="section-one-left">
-                    <h1>Your data,</h1>
-                    <h1>where you <span style={{ whiteSpace: "nowrap" }}>want it,</span></h1>
-                    <h1>in <span style={{ wordBreak: "break-word", hyphens: "auto" }}>milliseconds</span></h1>
+                    <h1>Real-time ETL & DCD</h1>
```
Comment (jshearer) [inline src/components/SectionOne.tsx:112]: `DCD` -> `CDC`

**Review by jshearer — CHANGES_REQUESTED** [review]
Looks great other than the 1 typo that I could see. Loving the `showTour` flag to `<Layout>` and new `<HubspotModal/>` component 😍

File: src/components/SectionOne.tsx (hunk 1 above)
Comment (flavioriper) [reply src/components/SectionOne.tsx]: Thanks, fixed

**Review by jshearer — APPROVED** [review]
LGTM thank you!

**Pull request merged**